result/arrow-parens.js

```diff
@@ -0,0 +1,94 @@
+/*eslint-env es6*/
+
+// Bad
+a => {}
```

Comment:
__[development][eslint]__
* Error - 'a' is defined but never used (no-unused-vars)